docs/remoteExams.md

```diff
@@ -8,6 +8,8 @@ This page lists sample assessment configurations for remote exams, where student
 
 ## Synchronous, timed exams
 
+This is the **recommended configuration**.
```

Comment:
How about the royal "We" to imply PL developer preference instead of the impersonal *this*? 

```suggestion
**We recommend exams be run using the synchronous timed configuration.** Below is an example of an assessment configured for students taking the exam at the same time with a time limit. 
```